src/hooks/useSession.ts

```diff
@@ -2,22 +2,24 @@
 import { useEffect, useState } from 'react';
 import { v7 as uuidv7 } from 'uuid';
 
-export const useSession = (): string | undefined => {
-  const [session, setSession] = useState<string | undefined>(() => {
+export const useSession = (): string => {
+  const [session, setSession] = useState<string>(() => {
     if (
       typeof window !== 'undefined' &&
       typeof sessionStorage !== 'undefined'
     ) {
-      return sessionStorage.getItem('session_id') || undefined;
+      return sessionStorage.getItem('session_id') || uuidv7();
     }
-    return undefined;
+    return uuidv7();
```

Comment:
why?

Reply:
to ensure we always have a value in session